### PR TITLE
Add univalue/1.0.5 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **univalue/1.0.5**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/univalue/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self.settings):
+                env = {
+                    "CC": "cl -nologo",
+                    "CXX": "cl -nologo",
+                    "CPP": "cl -nologo -EP",
+                    # "CPP": "{} cl -nologo -EP".format(self.deps_user_info["automake"].compile),
+                    "LD": "link",
+                    "CXXLD": "link",
+                    "AR": "{} lib".format(tools.unix_path(self.deps_user_info["automake"].ar_lib)),
```
Comment (madebr) [inline recipes/univalue/all/conanfile.py]: Depends on #820

File: recipes/univalue/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                }
+                with tools.environment_append(env):
+                    yield
+        else:
+            yield
+
+    def build(self):
+        self._patch_sources()
+        with tools.chdir(self._source_subfolder):
+            self.run("{} --verbose --install --force".format(os.environ["AUTORECONF"]), win_bash=tools.os_info.is_windows)
```
Comment (madebr) [inline recipes/univalue/all/conanfile.py]: Depends on #659

**Comment by madebr** [comment]
@uilianries @danimtb The bot is misleading me.

**Comment by danimtb** [comment]
@madebr let me relaunch the CI and see what is the final decision of the bot: 👍 or 👎 ? 🤣

**Comment by uilianries** [comment]
That bot decides the CI result by rolling a dice :game_die:

**Comment by danimtb** [comment]
Indeed, libtool is still not in the central repo. Waiting for #659 to be merged

**Comment by madebr** [comment]
@danimtb You can remove `blocked`, me think.

**Comment by madebr** [comment]
Sorry, needs #1102

**Comment by danimtb** [comment]
Relaunching CI here as #1102 was merged

**Comment by madebr** [comment]
This (probably) needs #807

File: recipes/univalue/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
```
Comment (uilianries) [inline recipes/univalue/all/conanfile.py:91]:
```suggestion
            self.run("{} --verbose --install --force".format(tools.get_env("AUTORECONF")), win_bash=tools.os_info.is_windows)
```

File: recipes/univalue/all/test_package/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
```
Comment (uilianries) [inline recipes/univalue/all/test_package/CMakeLists.txt:4]:
```suggestion
```

**Pull request merged**